src/layouts/EditHomepage/constants.ts

```diff
@@ -1,9 +1,44 @@
+import moment from "moment"
+
 export const RESOURCES_SECTION = {
   title: "Resources",
   subtitle: "Add a preview and link to your Resource Room",
   id: "resources",
 } as const
 
+export type AnnouncementSectionType = {
```

Comment:
clarifying your intent here - the type here is nearly a constant, with the **only** possible change being `date`. this means that, for example, `title` cannot be a `string` - it has to be the _constant_ string "New Announcement". if this is intended then mark this as resolved, otherwise we might benefit from giving a more relaxed type (eg: `string`)

Reply:
hey yup you are right, relaxed [here](70c135bee8c2cf96af40ded17b92e980c3eab16d)